src/js/edu-benefits/components/veteran-information/VeteranInformationFields.jsx

```diff
@@ -24,9 +26,12 @@ export default class PersonalInformationFields extends React.Component {
               month={this.props.data.veteranDateOfBirth.month}
               year={this.props.data.veteranDateOfBirth.year}
               onValueChange={(update) => {this.props.onStateChange('veteranDateOfBirth', update);}}/>
-          <Gender
+          <ErrorableRadioButtons
```

Comment:
Hm good question. I would keep it as you have it so we can keep the dropdown for HCA (although not sure why we need a dropdown for HCA... maybe something to revisit later :)).